test/stub/keyring-bridge.js

```diff
@@ -0,0 +1,55 @@
+export const KNOWN_PUBLIC_KEY =
+  '02065bc80d3d12b3688e4ad5ab1e9eda6adf24aec2518bfc21b87c99d4c5077ab0';
+
+export class FakeKeyringBridge {
+  #publicKeyPayload;
+
+  constructor({ publicKeyPayload }) {
+    this.#publicKeyPayload = publicKeyPayload;
+  }
+
+  async init() {
+    return Promise.resolve();
+  }
+
+  async dispose() {
+    return Promise.resolve();
+  }
+
+  async destroy() {
```

Comment:
I see that the `destroy` and `updateTransporMethod` methods are present in the `ledger-bride` but not in the `trezor-bridge`. Could it be this something specific to Ledger class? If so, maybe this should be moved to the child Ledger class?

Reply:
Good point! Ideally, all bridges should have `init` and `destroy` methods, but since we don't have a shared `KeyringBridge` type there are still some unwanted differences in method names - which were fixed on Keyring level, but Bridges are still different.

That being said, we can move `destroy` and `dispose` to the specific Trezor and Ledger fake bridge classes for now

Reply:
Refactored in [58d9a57](https://github.com/MetaMask/metamask-extension/pull/25824/commits/58d9a57bba2d245bd5a6d7d0137674b2cc80d2ee)